drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -535,9 +546,19 @@ static int hf_clock_enable(bool on, bool blocking)
 		return ret;
 	}
 
+	if (on) {
+		++clock_requested;
+	} else {
+		--clock_requested;
```

Comment:
`clock_requested` will never be greater then 1 and lower the 0, why not declare it as bool and assign variable `on `to it?

Reply:
Changed.